.codeclimate.yml

```diff
@@ -0,0 +1,19 @@
+---
+engines:
+  eslint:
+    enabled: true
+    channel: "eslint-2"
+  duplication:
+    enabled: true
+    checks:
+      Similar code:
+        enabled: false
+    config:
+      languages:
+      - javascript
+ratings:
+  paths:
+  - src/*.js
+exclude_paths:
+- src/js/*.js
```

Comment:
Is there a reason you are excluding your Javascript files from CodeClimate's ratings here?